scss/components/_icons.scss

```diff
@@ -114,3 +99,50 @@ $logos-event: url(../assets/event-logo-scaled.png);
   height: 64px;
   background-size: 100%;
 }
+
+@include desktopResolution {
+  %large-logo-size {
+    background-image: $logos-sheet-path;
+    width: 136px;
+    height: 32px;
+    display: block;
+    background-size: 100%;
+    opacity: 0.5;
+  }
+
+  .logo {
+    &-1 {
+      @extend %large-logo-size;
+    }
+
+    &-2 {
+      @extend %large-logo-size;
+    }
+
+    &-3 {
+      @extend %large-logo-size;
+    }
+
+    &-4 {
+      @extend %large-logo-size;
+    }
+
+    &-5 {
+      @extend %large-logo-size;
```

Comment:
- [ ] **OPTIONAL**
For best practices, please follow the DRY (don't repeat yourself)
This could be shrunk in 
```
&-1,
&-2,
.. etc {
// code here
}
```

Reply:
Thanks! you are right! don't know how I miss that!!